debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/ValueTypeHelper.java

```diff
@@ -218,7 +219,10 @@ public J9ClassPointer findJ9ClassInFlattenedClassCacheWithSigName(J9ClassPointer
 		@Override
 		public boolean isRomClassAValueType(J9ROMClassPointer romClass) throws CorruptDataException {
 			long flag = J9JavaAccessFlags.J9AccClassHasIdentity;
-			return (flag != 0) && !romClass.modifiers().anyBitsIn(flag);
+			return (flag != 0) /* Always return false if identity flag isn't defined. */
+				&& (J9Consts.VALUE_TYPES_MAJOR_VERSION <= romClass.majorVersion().longValue())
+				&& (J9Consts.PREVIEW_MINOR_VERSION == romClass.minorVersion().longValue())
+				&& !romClass.modifiers().anyBitsIn(flag);
```

Comment:
The version check here is `(major >= 23) && (minor == 0xFFFF)`; should it be `(major > 23) || ((major == 23) && (minor == 0xFFFF))`? Or do we need to wait to update the version boundary when value types are no longer a preview feature?

Reply:
I think it makes sense to wait, its not known at this time when this feature will be released as a preview feature or as a permanent feature.

Reply:
Is there an issue where we could include a reminder to revisit this? If not, could you please create one?